server/config/default.json

```diff
@@ -11,7 +11,8 @@
   },
   "database": {
     "file": "./messages.db",
-    "type": "sqlite3"
+    "type": "sqlite3",
+    "aliasRefreshInterval": "0 1 3 * * *"
```

Comment:
any reason to change the default from twice an hour?

Reply:
Id also like this left at twice an hour by default

Reply:
ah, that was a copy and paste error. default should be "0 5,35 * * * *", i believe i fixed it